_conferences/droidcon-pl-2015.md

```diff
@@ -0,0 +1,10 @@
+---
+name: "Droidcon"
+website: http://droidcon.pl
+location: Kraków, PL
+
+date_start: 2015-12-04
+date_end:   2015-12-06
+
+cfp_site: http://goo.gl/forms/mfJxBfi7L3
```

Comment:
Just put it as starting yesterday and ending 4 weeks before the conference. If the end date changes, we can update the site.